frontend/src/components/help-panel/HelpPanel.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  createContext,
+  Dispatch,
+  FunctionComponent,
+  ReactElement,
+  SetStateAction,
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+} from 'react'
+
+type PanelElement = string | ReactElement
+type PanelContext = {
+  element: string | ReactElement
```

Comment:
question: why is `string` a valid content type?

Reply:
it was put in place for testing purposes - it's simpler to give a string to the `tools` then a full `HelpPanel` - but it's not needed anymore, I will remove it, thank you